src/infection-model/index.tsx

```diff
@@ -178,7 +178,10 @@ export function curveInputsWithRt(
   return curveInputs;
 }
 
-function prepareCurveData(inputs: CurveFunctionInputs): CurveProjectionInputs {
+function prepareCurveData(
+  inputs: CurveFunctionInputs,
+  duration?: number,
```

Comment:
I think you can just pass `NUM_DAYS` as a [default argument](https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters) to `duration` here and you wouldn't need to reassign it to `numDays` below